src/card/CreditCard.java

```diff
@@ -482,5 +482,53 @@ public static void main(String[] args) {
 			e.printStackTrace();
 		}
 	}
+	public static boolean updateCreditCardInformation(String currentNickname, ConnectionSource databaseConnection, User safeStoreUser,String[] fieldsToModify, String[] newInputs) {
```

Comment:
maybe have this take in a Map<String, String>  instead of two arrays just bc it has so many parameters this will make it take one fewer

Reply:
I just deleted fieldsToModify and just checked if each newInputs field was empty or not in this method vs splitting it between the 2, that sound good?